src/ontology/uberon-edit.obo

```diff
@@ -224517,6 +224517,20 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8850000
+name: lumen of alveolus
+def: "An anatomical cavity that is part of the alveolus of the lung" [PMID:30390118]
+synonym: "Alveolar lumen" EXACT []
+synonym: "Cavity of pulmonary alveolus" EXACT [FMA:62779]
+synonym: "Lumen of pulmonary alveolus" EXACT [FMA:62779]
+synonym: "Pulmonary alveolar cavity" EXACT [FMA:62779]
+intersection_of: UBERON:0000464 ! anatomical space
+intersection_of: luminal_space_of UBERON:0002299 ! alveolus of lung
+relationship: dc-contributor https://orcid.org/0009-0000-8480-9277
+relationship: part_of UBERON:0002299 ! alveolus of lung
```

Comment:
This relationship is redundant, since luminal_space_of is a subclass of part_of.